src/field_screen_effect.c

```diff
@@ -1342,6 +1343,21 @@ enum {
     FRLG_WHITEOUT_HEAL_SCRIPT,
 };
 
+static const u8* GenerateRecoveryMessage(u8 taskId)
+{
+    bool32 forfeitTrainer = DidPlayerForfeitNormalTrainerBattle();
+    bool32 destinationIsPlayersHouse = (gTasks[taskId].tIsPlayerHouse == TRUE);
```

Comment:
Do we need this instead of just checking `gTasks[taskId].tIsPlayerHouse`?

Reply:
Just makes the next few lines more verbose - I don't feel strongly.